src/DynamoCoreWpf/ViewModels/Core/GalleryViewModel.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Dynamo.Core;
+using Dynamo.Interfaces;
+using Dynamo.UI.Commands;
+using Dynamo.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml.Serialization;
+
+namespace Dynamo.Wpf.ViewModels.Core
+{
+    public class GalleryContent : NotificationObject
+    {
+        private string header;
+        public string Header { get; set; }
+
+        private string body;
+        public string Body { get; set; }
+
+        private string image;
+        public string Image { get; set; }
+
+        private bool isCurrent;
+        public bool IsCurrent
+        {
+            get
+            {
+                return isCurrent;
+            }
+            set
+            {
+                isCurrent = value;
+                RaisePropertyChanged("IsCurrent");
+            }
+        }
+    }
+
+    public class GalleryContents
+    {
+        public List<GalleryContent> GalleryUIContents { get; set; }
+
+        public static GalleryContents Load(string filePath)
+        {
+            var galleryContents = new GalleryContents();
```

Comment:
Should move this `GalleryContents galleryContents = null` inside the `try` block, then return at the end of `try` block, in `catch` we should return `new GalleryContents()`;